app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
```

Comment:
- [ ] great job creating the `Posts` controller with the `index` action. However, you seem to have missed the `show` action`, please add it to your controller file. 🙏